nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -1131,21 +1147,33 @@ def from_tensorflow(self, graph, layout="NHWC"):
                     node.input[0] = in_name
 
                 # Fill shapes for all inputs in a list
-                try:
-                    inputs = [self._nodes[i] for i in node.input]
-                    for i in node.input:
+                inputs = []
+                for i in node.input:
+                    if i in self._nodes:
+                        inputs.append(self._nodes[i])
```

Comment:
I found the case that causes KeyError in `self._nodes[i]` in [my current work.](#2001 ) I think that it is needed to handle `KeyError`. I'd appreciate your help.

For example: 
```
    def check_split_concat(ishape, **kwargs):
        inp_array = np.random.uniform(size=ishape).astype(np.float32)
        with tf.Graph().as_default():
            in1 = tf.placeholder(shape=inp_array.shape, dtype=inp_array.dtype)
            splited = tf.split(in1, **kwargs)
            tf.concat(splited, axis=1)
            compare_tf_with_tvm(inp_array, 'Placeholder:0', 'concat:0')

    check_split_concat((5, 30), num_or_size_splits=[15, 15], axis=1)
```
In above code, the concat node has three inputs(`split`, `split1`, `concat/axis`), but `split1` is not registered in _nodes.(the detail of graphdef in this case is [here](dabff8cb4b54302ccc8d269e49baea10)) Moreover, these `split:X` is increased depending on the number of split.

Reply:
Please don't care above my comment. I understand it is a problem which should be resolved separately from this PR. Thanks ;)